docs/developer/extending/apps/architecture/app-permissions.mdx

```diff
@@ -54,6 +54,12 @@ const Component = () => {
 
 Once the user approves the permission request, the app receives the granted permissions. The user is then redirected to the `appUrl` defined in the app's manifest, with a `redirectUrl` query parameter appended. This parameter's value corresponds to the `redirectUrl` provided in `actions.RequestPermissions` (e.g., `redirectUrl=/success`). If the operation encounters an error, the `?error=REASON` query parameter will be appended to the redirect URL.
 
+:::warning
+
+Apps cannot be given `MANAGE_STAFF` permission. We chose this design because of the security implications - for example, an app could create a new staff user with administrator rights, outreaching its own permissions.
```

Comment:
```suggestion
Apps cannot be given `MANAGE_STAFF` and `MANAGE_APPS` permissions. We chose this design because of the security implications - for example, an app could create a new staff user with administrator rights, outreaching its own permissions, or create the app to act as one
```